src/Umbraco.Web.Website/ActionResults/RedirectToUmbracoPageResult.cs

```diff
@@ -135,5 +145,6 @@ public Task ExecuteResultAsync(ActionContext context)
             return Task.CompletedTask;
         }
 
+        private static bool IsUrlAbsolute(string url) => url.Contains("//");
```

Comment:
This would still see `~/test//double-leading/slash` as an absolute path and return it unchanged (making it an invalid redirect URL). Testing for absolute paths should be done using [`Uri.IsWellFormedUriString(url, UriKind.Absolute)`](https://docs.microsoft.com/en-us/dotnet/api/system.uri.iswellformeduristring?view=net-5.0).

With the above changes, this method can be removed though:

```suggestion
```